vsphere/internal/helper/virtualmachine/virtual_machine_helper.go

```diff
@@ -543,6 +543,30 @@ func Clone(c *govmomi.Client, src *object.VirtualMachine, f *object.Folder, name
 	return FromMOID(c, result.Result.(types.ManagedObjectReference).Value)
 }
 
+// InstantClone wraps the creation of a virtual machine instantly and the subsequent waiting of
+// the task. A higher-level virtual machine object is returned.
+func InstantClone(c *govmomi.Client, src *object.VirtualMachine, f *object.Folder, name string, spec types.VirtualMachineInstantCloneSpec, timeout int) (*object.VirtualMachine, error) {
+	log.Printf("[DEBUG] Instant Cloning virtual machine %q", fmt.Sprintf("%s/%s", f.InventoryPath, name))
```

Comment:
Since the Folder pointer and the source VM name  are used only for logging and not for the logic inside the function  the logging should be done outside the function call / the logging should be changed not to use these variables